src/DocumentsRepository/Traits/HTTPRepositoryTrait.ts

```diff
@@ -154,6 +161,21 @@ export const HTTPRepositoryTrait:{
 				} );
 		},
 
+		execute( this:HTTPRepositoryTrait, uri:string, requestOptions?:RequestOptions ):Promise<JSON> {
+			if( !this.context.registry.inScope( uri, true ) ) return Promise.reject( new IllegalArgumentError( `"${ uri }" is out of scope.` ) );
+			const url:string = this.context.getObjectSchema().resolveURI( uri, { base: true } );
+
+			return RequestService
+				.get( url, requestOptions )
+				.then( ( response:Response ) => {
+					return JSON.parse(response.data);
```

Comment:
Here we're returning a simple JSON object with the data retrieved from the platform. However, we might want to consider actually parsing the data and wrapping it as an object contained in the SDK's registry, just as it's done in the `executeSELECTQuery` method. Not wrapping the query result as a Document object has it's advantages to improve performance, but also prevents users from using added functionality within the objects they're retrieving. Overall, I think we can even provide two methods, one that returns a RAWSPARQLResult, just like it's done in the `executeRAWSELECTQuery` and another one that returns a decorated result just like the `executeSELECTQuery`. That provides users with the ability to select the option that works better for their solution.